apiserver/facades/client/spaces/spaces_mock_test.go

```diff
@@ -0,0 +1,350 @@
+// Copyright 2015 Canonical Ltd.
```

Comment:
I think there was some confusion here. I meant for the mocks to be generated into a file with this name (spaces_mock_test.go) the destination package being "spaces". This is not crucial, though if we keep the separate package, just generate all the mocks into a single file. Something like:
```go
//go:generate mockgen -package mocks -destination mocks/package_mock.go github.com/juju/juju/apiserver/facades/client/spaces Backing BlockChecker Machine
```
(Machine should be relocated to the spaces package)

The hand-rolled mocks here should just be generated mocks. If we're transitioning, there's no point in adding more hand-rolled ones.

The tests can live in the same test file as the others, we just add a new suite that we will migrate tests to over time.